benchmark/operations/create_dataset.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Operation to create a dataset.
+"""
+
+from functools import partial
+from uuid import uuid4
+
+from pyrsistent import PClass, field
+from zope.interface import implementer
+
+from flocker.common import loop_until
+
+from .._interfaces import IProbe, IOperation
+
+
+class EmptyClusterError(Exception):
+    """
+    Exception indicating that the cluster contains no nodes.
+    """
+
+
+@implementer(IProbe)
+class CreateDatasetConvergenceProbe(PClass):
+    """
+    Probe to create a dataset and wait for cluster to converge.
```

Comment:
It doesn't really wait for the cluster to converge in general, so maybe "create a dataset in the configuration and wait for the dataset to be created and mounted"?